program/src/processor.rs

```diff
@@ -6852,12 +6854,21 @@ pub fn read_oracle(
             let price_account = pyth_client::load_price(&oracle_data).unwrap();
             let value = I80F48::from_num(price_account.agg.price);
 
+            #[cfg(not(feature = "devnet"))]
+            let pyth_conf_filter = if oracle_ai.key
+                == &Pubkey::from_str("5bmWuR1dgP4avtGYMNKLuxumZTVKGgoN2BCMXWDNL9nY").unwrap()
```

Comment:
this looks like the right oracle for LUNA/USD: https://pyth.network/markets/#Crypto.LUNA/USD